rules/linux/lnx_file_and_directory_discovery.yml

```diff
@@ -0,0 +1,29 @@
+title: File and Directory Discovery
+id: d3feb4ee-ff1d-4d3d-bd10-5b28a238cc72
+status: experimental
+description: Detects usage of system utilities to discover files and directories
+author: Daniil Yugoslavskiy, oscd.community
+date: 2020/10/19
+references:
+  - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1083/T1083.md
+logsource:
+  category: process_creation
+  product: linux
+detection:
+  file_with_asterisk:
+    Image|endswith: '/file'
+    CommandLine|re: '(.){200,}' # execution of the 'file */* *>> /tmp/output.txt' will produce huge commandline
+  recursive_ls:
```

Comment:
There's a lot of legitimate activity that uses ls -R and find in Linux envs. I think it's quite hard to gain value from detections of this commands.

Reply:
discussion here https://github.com/SigmaHQ/sigma/pull/1362#discussion_r598336458